clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_utils.py

```diff
@@ -168,31 +167,20 @@ def ants_combine_transform(fix_image, moving_image, ants_warp_affine):
     out_warped = os.path.abspath("out_warped.nii.gz")
 
     cmd = (
-        f"antsApplyTransforms -o [out_warp_field.nii.gz,1] -i {moving_image} -r {fix_image} "
-        f"-t {ants_warp_affine[0]} {ants_warp_affine[1]} {ants_warp_affine[2]}"
+        f"antsApplyTransforms -o [{out_warp_field},1] -i {moving_image} -r {fix_image} "
+        f"-t {ants_warp_affine[0]} -t {ants_warp_affine[1]} -t {ants_warp_affine[2]}"
     )
     os.system(cmd)
 
     cmd1 = (
-        f"antsApplyTransforms -o out_warped.nii.gz -i {moving_image} -r {fix_image} "
-        f"-t {ants_warp_affine[0]} {ants_warp_affine[1]} {ants_warp_affine[2]}"
+        f"antsApplyTransforms -o {out_warped} -i {moving_image} -r {fix_image} "
+        f"-t {ants_warp_affine[0]} -t {ants_warp_affine[1]} -t {ants_warp_affine[2]}"
```

Comment:
Note for later, use nipype's interface for `antsApplyTransforms` instead of unsafe `os.system` calls.

Reply:
Could be done in a follow-up PR if time is a constraint.

Reply:
If the interface exists, we ought to use it directly.